types/numeral/index.d.ts

```diff
@@ -55,7 +55,7 @@ type RegisterType = 'format' | 'locale';
 interface Numeral {
     (value?: any): Numeral;
     version: string;
-    isNumeral: boolean;
+    isNumeral(value?: any): boolean;
```

Comment:
```suggestion
    isNumeral(value: any): boolean;
```
should it be really optional? (in definition type, I understand that JS works anway here, but implementaton does not provide defualt value when param is missing, it works because of language detail (undefined is not instance of T)

Reply:
@peterblazejewicz It doesn't have to, I wasn't sure what we would be the conventional way, your explanation makes sense.
Thank you.